Core/include/Acts/Seeding/GNN_TrackingFilter.hpp

```diff
@@ -229,8 +229,8 @@ class TrigFTF_GNN_TrackingFilter {
     const float weight_x = 0.5;
     const float weight_y = 0.5;
 
-    const float maxDChi2_x = 60.0;  // 35.0;
-    const float maxDChi2_y = 60.0;  // 31.0;
+    const float maxDChi2_x = 60.0;  // was 35.0;
+    const float maxDChi2_y = 60.0;  // was31.0;
```

Comment:
```suggestion
    const float maxDChi2_y = 60.0;  // was 31.0;
```

Reply:
Sorry, I just realised this is still a draft

Reply:
yes sorry still need to fix some formatting, thank you for pointing this one out!